src/databricks/labs/remorph/reconcile/schema_compare.py

```diff
@@ -0,0 +1,126 @@
+import logging
+from dataclasses import asdict
+
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql.types import BooleanType, StringType, StructField, StructType
+from sqlglot import parse_one
+
+from databricks.labs.remorph.config import MorphConfig
+from databricks.labs.remorph.reconcile.constants import SourceType
+from databricks.labs.remorph.reconcile.recon_config import (
+    Schema,
+    SchemaMatchResult,
+    SchemCompareOutput,
+    Table,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class SchemaCompare:
+    def __init__(
```

Comment:
```suggestion
    def __init__( self, table_conf, spark) 
```
Move the rest of the parameters to the individual methods.